docs-sources/includes/_Time.md

```diff
@@ -0,0 +1,33 @@
+Time manipulation
+=================
+
+<aside class="warning">
+When running MAGE in a cluster, make sure all the server's clocks
+are synchronised! MAGE is not responsible for maintaining clock
+synchronicity between servers
+</aside>
+
+> lib/index.js
+
+```javascript
+// TODO
+```
+
+Some types of games are meant to be played over fixed period of time;
+some game features may also be time-sensitive. For instance, you might want
+to give daily bonuses on player log in.
+
+However, during testing, you probably do not want to wait for a whole day to
+see if your player will be awarded a bonus. To deal with this issue, you can
+use the MAGE built-in `mage.time` module. This module allow developer
+to slow down or accelerate time from the MAGE process' perspective.
+
+This feature is often refered to as *time bending*
+
+Note that this does not affect existing APIs (such as `setTimeout`,
+`setInterval`, or the `Date` class); instead, you will need to use the
+the time module to compute a time value from the server's perspective,
+then use that value as a setTimeout/setInterval/new Date call argument.
```

Comment:
It's just a suggestion, since it makes it easier to tie the `mage.time.msec()` function. For what it's worth by the way, the `safe-timers` module is authored by Wizcorp.